alica_engine/src/engine/TeamObserver.cpp

```diff
@@ -124,7 +122,8 @@ void TeamObserver::close()
  */
 void TeamObserver::doBroadCast(const IdGrp& msg) const
 {
-    if (!_maySendMessages) {
+    bool maySendMessages = !_config["Alica"]["SilentStart"].as<bool>();
```

Comment:
- need a reload function for the config
- can load this value whenever the config changes instead of doing it here

Reply:
ok